runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -789,6 +827,18 @@ public void onGraphicChanged(GraphicChanged event)
 		}
 	}
 
+	// compare current inv state with previous inv state
+	private int getPreviousItemId(Item[] prevInv, Item[] newInv) {
+		for (int i = 0; i < prevInv.length; i++) {
+			if (ANTIPOISON_ITEMS.contains(prevInv[i].getId())) {
+				// did it change?
+				if (prevInv[i].getId() != newInv[i].getId() || newInv[i] == null) return prevInv[i].getId();
+			}
+		}
+
+		return -1;
+	}
+
 	/**
 	 * remove SOTD timer when weapon is changed
```

Comment:
This comment will need to be updated. (Removed is probably the better approach, since it doens't really make much sense to document a subscriber)